examples/oft-evm-move-adapters/sources/oft_implementation/move_oft_adapter.move

```diff
@@ -76,9 +87,11 @@ module oft::move_oft_adapter {
         fa: &mut FungibleAsset,
         min_amount_ld: u64,
         dst_eid: u32,
-    ): (u64, u64) acquires OftImpl {
+    ): (u64, u64) acquires OftImpl, PauserStore {
         assert_not_blocklisted(sender);
         assert_metadata(fa);
+        // Global outflow pause gate
+        assert!(!is_paused(), EPAUSED);
```

Comment:
Think this should also be added to credit so incoming bridges would also be blocked.